design/pipeline-artifacts.md

```diff
@@ -16,8 +18,8 @@ Artifacts are distinct from other `resources` types, including `containers`, `re
 
 ```yaml
 - upload: string # a minimatch path or list of [minimatch paths](tasks/file-matching-patterns.md) to upload
-  artifact: string # identifier for this artifact (no spaces allowed), defaults to 'default'
-  prependDirectory: string # a directory path that will be prepended to all uploaded files
+  artifact: string # identifier for this artifact (no spaces allowed), defaults to the job name
```

Comment:
The job name is typically `__default`. Phase name is also problematic because it doesn't disambiguate the job configuration.

Consider something like: if job name is __default, use `<phase>`, otherwise use `<phase>.<job>`

Reply:
Good call, thanks.